testing/integration/ess/otel_test.go

```diff
@@ -1648,13 +1648,6 @@ func AssertMapsEqual(t *testing.T, m1, m2 mapstr.M, ignoredFields []string, msg
 	flatM1 := m1.Flatten()
 	flatM2 := m2.Flatten()
 	for _, f := range ignoredFields {
-		hasKeyM1, _ := flatM1.HasKey(f)
-		hasKeyM2, _ := flatM2.HasKey(f)
-
-		if !hasKeyM1 && !hasKeyM2 {
-			assert.Failf(t, msg, "ignored field %q does not exist in either map, please remove it from the ignored fields", f)
-		}
-
```

Comment:
I feel that this just disappearing might lead us to losing track of having to do that and the reasons behind it, so wdyt if we instead comment these out and put a link to the Flaky github issue that has the necessary context of why or how this is happening? 🙂